storage-stack-automation/README.md

```diff
@@ -0,0 +1,36 @@
+# Automation using Lambda to Deploy FSS- Storage Stack 
+
+Automated process to deploy FSS Storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined.
```

Comment:
Automated process to deploy an FSS storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined.

Reply:
fixed